core/sql/common/ComMisc.cpp

```diff
@@ -99,3 +99,17 @@ NABoolean ComIsTrafodionReservedSchemaName(
   return FALSE;
 }
 
+// schema names of pattern "_HV ... _" and "_HB_ ... _" are reserved to store
```

Comment:
The ... was supposed to signify the schema name, perhaps "_HV_schema_" and "_HB_schema_" would look better.  The ... to indicate the schema name may be confusing.